diagnostic.md

```diff
@@ -36,13 +39,17 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://localhost:4741/donuts/1
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let id = 1;
+let getDonut = $.ajax({
+    url: app.host + '/donuts/' + id,
+    method: 'GET',
+  });
 ```
```

Comment:
```js
let getDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'GET',
  });
};
``` It makes more sense to assign the id this way, so we dont need to reassign it's value everytime we use the function.